pkgs/top-level/python-packages.nix

```diff
@@ -25797,4 +25797,21 @@ in modules // {
     };
   };
 
+  topydo = buildPythonPackage rec {
+    name = "topydo-${version}";
+    version = "0.9";
+    disabled = (!isPy3k);
+
+    src = pkgs.fetchFromGitHub {
```

Comment:
oh, I hadn't checked that myself. No, that is not possible.
Please do ask upstream to upload source tarballs in the future.